packages/iTwinUI-react/src/core/Menu/MenuItemSkeleton.tsx

```diff
@@ -0,0 +1,83 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { CommonProps, useTheme } from '../utils';
+import '@itwin/itwinui-css/css/menu.css';
+
+export type MenuItemSkeletonProps = {
+  /**
+   * Flag whether to show skeleton for sub-label.
+   */
+  hasSublabel?: boolean;
+  /**
+   * Flag whether to show skeleton for icon.
+   */
+  hasIcon?: boolean;
+  /**
+   * Skeleton content width in percentage.
+   */
+  contentWidth?: number;
+  /**
+   * Translated strings used for accessibility.
+   */
+  translatedStrings?: {
+    /**
+     * Label for loading state. Defaults to "Loading…".
+     */
+    loading: string;
+  };
+} & CommonProps;
+
+/**
+ * Loading skeleton menu item.
+ */
+export const MenuItemSkeleton = (props: MenuItemSkeletonProps) => {
+  const {
+    hasSublabel,
+    hasIcon,
+    contentWidth,
+    translatedStrings = { loading: 'Loading…' },
+    className,
+    style,
+    ...rest
+  } = props;
+
+  useTheme();
+
+  return (
+    <li
+      className={cx(
+        'iui-menu-item',
+        'iui-menu-item-skeleton',
+        { 'iui-large': hasSublabel },
+        className,
+      )}
+      style={{
+        ...{
+          '--iui-menu-item-content-skeleton-max-width': contentWidth
+            ? `${contentWidth}%`
+            : undefined,
+        },
+        ...style,
+      }}
+      {...rest}
+    >
+      {hasIcon && <div className='iui-icon iui-skeleton' aria-hidden='true' />}
```

Comment:
nit: you don't need `='true'` in react

Reply:
Removed

Reply:
Missed in one spot 😆 
![ ](https://user-images.githubusercontent.com/9084735/172391613-3ab3fedf-b2ff-4437-82d1-b4c24bd1e84e.png)

Reply:
Upsy, removed